flycheck.el

```diff
@@ -79,6 +79,7 @@
 (require 'help-mode)             ; `define-button-type'
 (require 'find-func)             ; `find-function-regexp-alist'
 (require 'json)                  ; `flycheck-parse-tslint'
+(require 'ansi-color)            ; strip ansi color from command output
```

Comment:
Please move this require into the body of the rebar3 `:error-parser` function.  That'll break the build due to a byte compiler warning, which we can then fix together.